execution-engine/src/hcall/buffer.rs

```diff
@@ -0,0 +1,138 @@
+//! The Veracruz host-call interface, and its implementation.
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Copyright
+//!
+//! See the file `LICENSE.markdown` in the Veracruz root directory for licensing
+//! and copyright information.
+//!
+///! GLUE CODE FOR NEW API.
+///! IT SHOULD BE REPLACED BY VFS IMPLEMENTATION OF WASI API.
+
+use err_derive::Error;
+use std::{collections::HashMap, result::Result, vec::Vec, string::{ToString, String}};
+use veracruz_utils::{VeracruzCapabilityIndex, VeracruzCapability, VeracruzCapabilityTable};
+use serde::{Deserialize, Serialize};
+
+/// Error type for mexico-city buffer.
+#[derive(Clone, Debug, Error, Serialize, Deserialize)]
+pub enum VFSError {
+    //TODO: potential remove this 
```

Comment:
Is this TODO still relevant?  In which case please open an issue to keep track of this.

Reply:
This file is glue code, and will be removed in wasi change. I have documented it in the beginning in case I forgot.